cloud/deployment/deployment.go

```diff
@@ -1294,6 +1276,92 @@ func updateWorkerQueuesForExecutor(newExecutor astroplatformcore.DeploymentExecu
 	}
 }
 
+func CreateDefaultTaskPodCPU(defaultTaskPodCPU string, isRemoteExecutionEnabled bool, configOption *astrocore.DeploymentOptions) *string {
```

Comment:
nit: should we add unit tests for these new functions? Or are we sure that the existing test cases cover all possible variations?